src/runner.js

```diff
@@ -17,6 +20,13 @@ const Runner = (config: Config, registry: Reg, logger: Object) => {
     for (const m of messages) { // eslint-disable-line
       let params = null;
       try {
+        if ( maxRss ) {
+          const currentRss = memoryUsage().rss;
+          if (currentRss > maxRss) {
```

Comment:
Maybe? I dunno. Kinda subjective I think. Semantically I think "If it's at the max, that's cool. If it's beyond the max, oh no cover your ears."

If you think it'd be clearer as >= though I'm very happy with that. Whatever the majority view is we should go with.